wfe/wfe.go

```diff
@@ -69,6 +69,10 @@ const (
 
 	// POST requests with a JWS body must have the following Content-Type header
 	expectedJWSContentType = "application/jose+json"
+
+	// deactivatedStatus is the only valid status accepted by ACME to update Accounts
+	// and auhtorizations.
+	deactivatedStatus = "deactivated"
```

Comment:
Could you add this to `acme/common.go` as a new top level exported status alongside `StatusPending`, `StatusValid`, etc? Only the `wfe` references it presently but I think its best to keep all of the ACME constants in one place where possible.